elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SubselectTest.java

```diff
@@ -87,7 +92,10 @@ public void testJoinToFilter() throws Exception {
                         PlayerStats.class, false))
                 .build();
 
-        List<Object> results = StreamSupport.stream(engine.executeQuery(query).getData().spliterator(), false)
+        FutureTask<QueryResult> queryResult = engine.executeQuery(query);
+        queryResult.run();
```

Comment:
Remove all references to queryResult.run() in this PR.

Reply:
The delta between the original line and your modifications can be just 4 characters:

```java
List<Object> results = StreamSupport.stream(engine.executeQuery(query).get().getData().spliterator(), false)
```